pages/index.js

```diff
@@ -26,7 +97,13 @@ class HomePage extends Component {
             </div>
           </div>
           <div className="countdownWrapper">
-            <Countdown />
+            <div className="mainContainer form-wrapper">
+              <Banner
+                {...this.props}
+                {...this.state}
```

Comment:
Have we tested how this looks with a poor internet connection? Might be worth to check if it's loading and/or add a default message like `"Loading"` or something.

Reply:
According to my tests on Chrome, by limiting the network speeds, it works fine.

The banner only needs the contents of `https://api.mybit.io/home` to work, which is a request made server side, for which case the network speeds of the client are not important.

Reply:
Fair enough, good for me